include/crypto/aes_engine.hpp

```diff
@@ -0,0 +1,167 @@
+#pragma once
+
+#include <array>
+#include <cstring>
+#include <cstdint>
+#include <climits>
+#include <filesystem>
+#include <optional>
+
+#include "helpers.hpp"
+
+namespace Crypto {
+	constexpr std::size_t AesKeySize = 0x10;
```

Comment:
`std::size_t` -> `usize`
`uint8_t` -> `u8`
etc

Reply:
I changed the uint8_t to u8 but the codebase doesn't use the `usize` type in helpers.hpp that much atm.
Maybe we should have a poll among contributors about it?